src/components/label/index.ts

```diff
@@ -0,0 +1,38 @@
+import Block from '../../core/block';
+import validation from '../../core/validation';
+import template from './label.hbs';
+
+interface LabelProps {
+  page: string;
+  type: string;
+  name: string;
+  value: string;
+  text: string;
+  required: string
```

Comment:
судя по названию, тот больше подходит тип ```boolean```
```suggestion
  required: boolean
```